src/PhpSpec/Listener/ClassNotFoundListener.php

```diff
@@ -75,7 +76,13 @@ public function afterSuite(SuiteEvent $event)
 
             if ($this->io->askConfirmation($message)) {
                 $this->generator->generate($resource, 'class');
-                $event->markAsWorthRerunning();
+                if (class_exists($classname)) {
+                    $event->markAsWorthRerunning();
+                }
+                else {
+                    $message = 'File was written but the class %s was not autoloadable - do you have an autoloader configured?';
```

Comment:
Is there more detail in the docs on this? If so, is it worth having the url for that in the message as well?